scripts/compare_test_signatures.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/sh
+
+set -e
+
+# compare test signatures generated:
```

Comment:
Do you think this is sufficient to ensure that these don't drift?

Perhaps we should also be checking the signatures generated in (1), (2) and (3) in (1) (2) and (3)

so like:

```
1 -> 2
1 -> 3
2 -> 1
2 -> 3
3 -> 1
3 -> 2
```

How hard would it be to transfer the binaries over so we could do this check?

Reply:
I think if we're testing whether the signatures are identical, it's enough to verify the signatures within 1, 2, and 3.

Reply:
as you said, since the signatures are the same, we only need to run verify 3 times

Reply:
Verifications added.